src/main/scala/se/nimsa/sbx/dicom/DicomUtil.scala

```diff
@@ -243,4 +244,8 @@ object DicomUtil {
     if (name == null) "" else name
   }
 
+  def padToEvenLength(bytes: ByteString): ByteString = {
+    val padding = if (bytes.length % 2 != 0) ByteString(0) else ByteString.empty
```

Comment:
E.g. dcm4che VR constructor (https://github.com/dcm4che/dcm4che/blob/master/dcm4che-core/src/main/java/org/dcm4che3/data/VR.java#L214) takes padding byte in as argument.